editor/modes/visual-editor/style.scss

```diff
@@ -284,6 +297,29 @@
 	}
 }
 
+// show a separate cursor/caret highlight style for when an item was selected by the mouse or keyboard (but isn't multi selected)
```

Comment:
How would this scale? Is this something a block author should be able to opt into? What are the reasons for a block to be part of this group?